ingenialink/canopen/register.py

```diff
@@ -67,14 +69,19 @@ def __init__(
         description: Optional[str] = None,
         default: Optional[bytes] = None,
         bitfields: Optional[dict[str, BitField]] = None,
+        monitoring: Union[tuple[None, None, None], tuple[int, int, RegCyclicType]] = (
```

Comment:
This format is new right? Are you sure you don't want to use a data type for storing this info?